### DISCUSSION TITLE
Chart & graph visualizations

### DISCUSSION THREAD

**Discussion by kgodey** [mathesar-foundation/mathesar#2284] [Feature Ideas]
We'd like to support the ability to display data in a table or exploration as a chart or graph.

---
*If you'd like us to build this feature, please comment below with more details about how you plan to use this functionality.*

**Comment by Hultner** [comment]
This would be really nice, it’s one of those features that DB Visualizer does really well but few other database IDEs. 
I use this a lot for fast ad-hoc charting of various stats to get a feeling for trends in the data.

**Reply by kgodey** [reply]
Thank you @Hultner! Could you provide any information on the types of charts and graphs you usually use?
